nexus/db-queries/src/db/datastore/volume.rs

```diff
@@ -4435,28 +4442,31 @@ mod tests {
         // need to add region snapshot objects to satisfy volume create
         // transaction's search for resources
 
-        let address_1 = String::from("[fd00:1122:3344:104::1]:400");
-        let address_2 = String::from("[fd00:1122:3344:105::1]:401");
-        let address_3 = String::from("[fd00:1122:3344:106::1]:402");
+        let address_1: SocketAddrV6 =
+            "[fd00:1122:3344:104::1]:400".parse().unwrap();
+        let address_2: SocketAddrV6 =
+            "[fd00:1122:3344:105::1]:401".parse().unwrap();
+        let address_3: SocketAddrV6 =
+            "[fd00:1122:3344:106::1]:402".parse().unwrap();
```

Comment:
wondered if it might be nicer to [programmatically construct these from an `Ipv6Addr` and port](https://doc.rust-lang.org/std/net/struct.SocketAddrV6.html#method.new) rather than parsing them, but honestly it's probably wordier than the current thing...